### ISSUE TITLE
USN-3438-1: Git vulnerability

### ISSUE THREAD

**Issue by Bregor** [deis/slugrunner#68]
Package `git-core` should be upgraded in `quay.io/deis/slugrunner` image due to [USN-3438-1](https://usn.ubuntu.com/usn/usn-3438-1/)

**Comment by bacongobbler** [comment]
I kicked off CI since heroku/cedar:14 was recently rebuilt. We are no longer vulnerable.

<img width="609" alt="git" src="https://user-images.githubusercontent.com/1360539/31237266-4db99cb4-a9ab-11e7-83c7-1324df00ee00.PNG">

**Comment by stuszynski** [comment]
Hi. I'm not so sure about this fix. 

```
$ docker run --entrypoint=dpkg deis/slugrunner:v2.4.0 --list git
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name                               Version                             Architecture Description
+++-==================================-===================================-============-===========================================================================================
ii  git                                1:1.9.1-1ubuntu0.5                  amd64        fast, scalable, distributed revision control system
```

In `deis/slugrunner:v2.4.0` there is a `git` package in `1:1.9.1-1ubuntu0.5` version installed. In the last relase of `heroku/cedar` there is `1:1.9.1-1​ubuntu0.6` version, but according to [usn-3438-1](https://usn.ubuntu.com/usn/usn-3438-1/) we need `1:1.9.1-1ubuntu0.7`([changelog](https://launchpad.net/ubuntu/+source/git/1:1.9.1-1ubuntu0.7)) to fix [CVE-2017-14867](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-1486). 

It's also related to https://github.com/deis/slugbuilder/issues/144

**Comment by bacongobbler** [comment]
Thank you @stuszynski. Would you mind filing a ticket upstream at https://github.com/heroku/stack-images to bump git? It does indeed appear that this has not been resolved. Thank you for double-checking.

**Comment by bacongobbler** [comment]
filed at https://github.com/heroku/stack-images/issues/83

**Comment by szymonpk** [comment]
Git was updated in the latest cedar.

**Comment by stuszynski** [comment]
@bacongobbler Would you kick off CI again?  I can see the last build was pushed 5 days ago but it still has ```git - 1:1.9.1-1ubuntu0.5```.

**Comment by bacongobbler** [comment]
fixed for slugrunner v2.4.0. I'm kicking off the pipeline for slugbuilder and will close this once that's done. Sorry for the delay!

```
><> docker run --entrypoint=dpkg deis/slugrunner:v2.4.0 --list git
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name                               Version                             Architecture Description
+++-==================================-===================================-============-===========================================================================================
ii  git                                1:1.9.1-1ubuntu0.7                  amd64        fast, scalable, distributed revision control system
```

**Comment by bacongobbler** [comment]
actually I'll close this and close https://github.com/deis/slugbuilder/issues/144 once slugbuilder's been fixed.
